src/app/cars/cars.component.html

```diff
@@ -0,0 +1,44 @@
+
+<h2>My cars</h2>
+<div class="d-inline-flex">
+<form>
+  <div class="form-group form-inline">
+      Full text search: <input class="form-control ml-2" type="text" [formControl]="filter"/>
+  </div>
+</form>
+
+<a routerLink="/new-car">
+  <td><button class="New Car ml-4" >New Car</button>
+  </a>
+</div>
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col" sortable="id" (sort)="onSort($event)">#</th>
```

Comment:
where is onSort() implemented?

Reply:
Some "leftover code" that doesn't belong here. Deleted.